index.js

```diff
@@ -1,28 +1,39 @@
-const readline = require('readline-sync')
+const prompts = require('prompts')
 const robots = {
   text: require('./robots/text.js')
 }
 
-async function start() {
-  const content = {}
+function askAndReturnAnswers() {
+  const questions = [
+    {
+        type: 'text',
+        name: 'searchTerm',
+        message: 'Type a Wikipedia search term:',
+        validate: value => typeof value === 'string' ? value.trim() !== '' : false,
+    },
+    {
+        type: 'select',
+        name: 'prefix',
+        message: 'Choose one option:',
+        choices: ['Who is', 'What is', 'The history of'],
+        validate: value => typeof value === 'string' ? value.trim() !== '' : false,
+    }
+  ]
+
+  return new Promise(async (resolve, reject) => {
+    const promptOptions = { 
+      onCancel: () => reject(new Error('The user has stopped answer'))
```

Comment:
Acredito que o mais correto seria `The user has stopped answering` o que acha @RodolfoSilva ?

Reply:
Obrigado, acho que ficou bem melhor :P